### DISCUSSION TITLE
Tile card should allow to set light color

### DISCUSSION THREAD

**Discussion by Nickduino** [home-assistant/frontend#22423] [Dashboard]
The Tile card allows to choose the brightness or the temperature of a light but not its color (even though the icon displays the current color - blue in this example): 

![image](https://github.com/user-attachments/assets/486a0719-f6b2-4e7e-9917-b407cfcf125f)

I believe "color temperature" should be renamed "temperature" and a new "color" option should be added

**Comment by rhysperry111** [comment]
I assume it'd make sense to have this as two sliders - hue and saturation.

It's honestly a huge gripe I've got with home assistant right now as I'm usually pretty hesitant to stray into custom cards and themes just in case things break with updates and whatnot.

**Comment by marcinbauer-ohf** [comment]
Adding a separate Hue and Saturation slider as a Feature could be of value. I wouldn't change the naming to just 'temperature' since in a home you can have temperature sensor read-outs and it could be confusing.

**Comment by Nickduino** [comment]
> I assume it'd make sense to have this as two sliders - hue and saturation.

Here is how the mushroom light card displays it:

![image](https://github.com/user-attachments/assets/e143033a-0f15-403b-af36-2fcd96ab5e0c)

There is no choice for saturation (not saying that's good, just stating how the card does it).

The other solution would be the a color wheel:

![image](https://github.com/user-attachments/assets/ec4bcfab-9ba5-4165-b2a3-53028674d72d)



> It's honestly a huge gripe I've got with home assistant right now as I'm usually pretty hesitant to stray into custom cards and themes just in case things break with updates and whatnot.

Agreed.


> I wouldn't change the naming to just 'temperature' since in a home you can have temperature sensor read-outs and it could be confusing.

"Light temperature" would make sense. It's not a sensor but the feature of a card, I don't think it's possible to confuse it for a sensor measuring the actual temperature of a light.

**Comment by Petro31** [comment]
2 Sliders, hue slider, saturation slider.  Users can expose both, one, or none.  I'd wager the majority of people would skip saturation completely.

**Reply by Nickduino** [reply]
> I'd wager the majority of people would skip saturation completely.

I have no idea what the majority of people would or wouldn't do (I, for one, like "pastel" colors rather than over-saturated ones) but at the moment, they can't even change the color, let alone the saturation.

**Comment by BBE-FR** [comment]
I personaly support the hue slider with maximum saturation if not already tuned differently (saturation can be tuned with the more info dialog box if needed) or the 2 slider as proposed above (and would bet that very few user will display both sliders).

Whatever the design, I would love to have the feature turning grey when light if off, contrarily to the light temperature feature that stays orange when light if OFF: [as reported here](https://github.com/home-assistant/frontend/issues/21895)

**Comment by silvanocerza** [comment]
For anyone interested in this I opened PR #25591 that added this as a feature but it has been rejected. 

Since I have no say in the project I decided to create a separate feature installable via HACS. You can find it [here](https://github.com/silvanocerza/light-card-hue-feature) with all instructions necessary.

**Comment by Nickduino** [comment]
> it has been rejected. 

Pity. They think the temperature is useful but not the colour, that makes no sense...

**Comment by rhysperry111** [comment]
Not sure if I entirely understand the argument since all this does is expose the *ability* to use hue as a feature and it's up to users to decide whether it's useful for their use-case.
